nwb_conversion_tools/utils.py

```diff
@@ -103,21 +103,27 @@ def get_schema_from_hdmf_class(hdmf_class):
         schema_arg = {docval_arg['name']: dict(description=docval_arg['doc'])}
 
         # type float
-        if docval_arg['type'] == 'float' or (isinstance(docval_arg['type'], tuple) and 'float' in docval_arg['type']):
+        if docval_arg['type'] == 'float' \
+            or (isinstance(docval_arg['type'], tuple)
+                and 'float' in docval_arg['type']):
             schema_arg[docval_arg['name']].update(type='number')
 
         # type string
-        elif docval_arg['type'] is str or (isinstance(docval_arg['type'], tuple) and str in docval_arg['type']):
+        elif docval_arg['type'] is str \
+            or (isinstance(docval_arg['type'], tuple)
+                and str in docval_arg['type']):
             schema_arg[docval_arg['name']].update(type='string')
 
         # type datetime
-        elif docval_arg['type'] is datetime or (isinstance(docval_arg['type'], tuple) and datetime in docval_arg['type']):
```

Comment:
This line was over 120 characters long, which is the standard over at `spikeextractors` and also what I use, so shortened it and aligned the nearby evaluations in a similar style.